jwt/circle.yml

```diff
@@ -0,0 +1,4 @@
+test:
+  post:
+    - if [ -e ./gradlew ]; then ./gradlew jacocoTestReport;else gradle jacocoTestReport;fi
```

Comment:
this file should not exist.
all configuration for circleCI goes into `.circleci/config.yml`